scrapy/extensions/feedexport.py

```diff
@@ -277,11 +277,25 @@ def close_spider(self, spider):
                         'itemcount': slot.itemcount,
                         'uri': slot.uri}
             d = defer.maybeDeferred(slot.storage.store, slot.file)
-            d.addCallback(lambda _: logger.info(logfmt % "Stored", log_args,
-                                                extra={'spider': spider}))
-            d.addErrback(lambda f: logger.error(logfmt % "Error storing", log_args,
-                                                exc_info=failure_to_exc_info(f),
-                                                extra={'spider': spider}))
```

Comment:
I wonder if [this](https://stackoverflow.com/a/2295372/939364) would work. It would be much cleaner.